followthemoney/export/rdf.py

```diff
@@ -14,10 +14,11 @@ def __init__(self, fh, qualified=True):
 
     def write(self, proxy, **kwargs):
         graph = Graph()
+
         for triple in proxy.triples(qualified=self.qualified):
             graph.add(triple)
         try:
             nt = graph.serialize(format="nt").strip()
-            self.fh.write(nt.decode("utf-8") + "\n")
+            self.fh.write(nt + "\n")
```

Comment:
`nt` defaults to `ascii` encoding. So this will break for unicode text. May be we should consider using `nt11` and add a test for unicode strings.

Reply:
Curious where you found nt11, doesn't seem to exist here: https://rdflib.readthedocs.io/en/stable/plugin_serializers.html?highlight=graph.serialize#id1 what am i missing?

Reply:
Seems like the docs are out of date. https://github.com/RDFLib/rdflib/blob/038af4547e799845e08986867b24acafcbe72d48/rdflib/plugin.py#L170 and https://github.com/RDFLib/rdflib/blob/038af4547e799845e08986867b24acafcbe72d48/rdflib/plugins/serializers/nt.py#L35